### ISSUE TITLE
[Bug] `beartype.claw` fails on empty modules containing only `from __future__` imports

### ISSUE THREAD

**Issue by kasium** [beartype/beartype#307]
If I use the beartype claw package on a bascially empty module with a annotations future import, it fails

Files:
**package.\_\_init\_\_**
```py
"""Some package"""
from __future__ import annotations
from beartype.claw import beartype_package
beartype_package("package")
```

**package.sub_package.\_\_init\_\_**
```py
"""Some package"""
from __future__ import annotations
```

Now execute `python -c "import package.sub_package"`

```
Traceback (most recent call last):
  File "/root/git/beartype/venv/lib/python3.11/site-packages/beartype/claw/_importlib/_clawimpload.py", line 477, in source_to_code
    module_codeobj = compile(
                     ^^^^^^^^
  File "/root/git/beartype/package/sub_package/__init__.py", line 3
    from __future__ import annotations
SyntaxError: from __future__ imports must occur at the beginning of the file

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<frozen importlib._bootstrap>", line 1178, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1149, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 936, in exec_module
  File "/root/git/beartype/venv/lib/python3.11/site-packages/beartype/claw/_importlib/_clawimpload.py", line 375, in get_code
    return super().get_code(fullname)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap_external>", line 1074, in get_code
  File "/root/git/beartype/venv/lib/python3.11/site-packages/beartype/claw/_importlib/_clawimpload.py", line 491, in source_to_code
    raise BeartypeClawImportAstException(
beartype.roar.BeartypeClawImportAstException: Module "package.sub_package" unimportable, as @beartype generated invalid abstract syntax tree (AST):

Module(
    body=[
        Expr(
            value=Constant(value='Some package')),
        ImportFrom(
            module='beartype._decor.decorcache',
            names=[
                alias(name='beartype', asname='__beartype__')]),
        ImportFrom(
            module='beartype.door._doorcheck',
            names=[
                alias(name='die_if_unbearable', asname='__die_if_unbearable_beartype__')]),
        ImportFrom(
            module='beartype.claw._clawstate',
            names=[
                alias(name='claw_state', asname='__claw_state_beartype__')]),
        ImportFrom(
            module='__future__',
            names=[
                alias(name='annotations')],
            level=0)],
    type_ignores=[])

ast.compile() exception (when passed the above AST):
        SyntaxError: from __future__ imports must occur at the beginning of the file (__init__.py, line 3)
```

Environment
* beartype: 0.16.4
* pyhton 3.11.0

**Comment by leycec** [comment]
**...lol.** So `beartype.claw` fails on empty modules containing only `from __future__` imports, huh? That's definitely *not* great. Although extremely uncommon and thoroughly useless, empty modules are still a valid use case. @beartype shouldn't be discriminating against them. *y u do this 2 us, @beartype!?*

Thanks so much for unveiling this high strangeness, @kasium. Together, we will build a better @beartype. Let's resolve this tonight for the good of Pythonistas from Germany everywhere. :bear:  :+1:

**Comment by leycec** [comment]
**Resolved by ff986d4a012b.** Until our upcoming @beartype 0.17.0 release drops, consider circumventing this on your end by just appending a trivial noop to semantically empty modules like this: e.g.,

```python
"""Some package"""
from __future__ import annotations

True  # <-- *TRIVIAL NOOP.* This is what I am saying.
```

Although technically stupid, this also pragmatically works around the issue at hand. In theory, Python *should* simply elide (i.e., compile away) trivial noops like that. Thus, nothing is lost but everything is gained.

Thanks yet again for this unbelievable yet true @beartype failure. This commit's for you! :muscle: :bear:

**Comment by kasium** [comment]
Thanks a lot for the fast fix!
